### PR TITLE
Add sample config file

### PR DESCRIPTION
**Pull request by stevenroose**

### CODE REVIEW

File: sample-lnd.conf [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+; Activate the debug htlc mode. With the debug HTLC mode, all payments sent use a
+; pre-determined R-Hash. Additionally, all HTLCs sent to a node with the debug
+; HTLC R-Hash are immediately settled in the next available state transition.
+debughtlc=false
+
+; Activate the hodl HTLC mode.  With hodl HTLC mode, all incoming HTLCs will be
+; accepted by the receiving node, but no attempt will be made to settle the
+; payment with the sender.
+hodlhtlc=false
```
Comment (Roasbeef) [inline sample-lnd.conf]: A comment should be added to detail that this option and the one above it exist primarily for testing purposes. 

In the case of `debughtlc` it uses the _same_ payment hash _each time_. This means that intermediate nodes that see this payment hash can instantly take the funds locked within the output during the clearing phase, skipping forwarding the HTLc to any other participants.

File: sample-lnd.conf [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+; The maximum number of incoming pending channels permitted per peer.
+maxpendingchannels=1
+
+; The default number of confirmations a channel must have before it's considered
+; open.
+defaultchanconfs=1
+                       
+; If true, then automatic network bootstrapping will not be attempted.
+nobootstrap=false
+; If set, wallet will be encrypted using the default passphrase.
```
Comment (Roasbeef) [inline sample-lnd.conf]: Minor nit: a new line should be added above.

File: sample-lnd.conf (hunk 1 above)
Comment (stevenroose) [reply sample-lnd.conf]: Maybe we should just skip the ones meant for testing? Testers know them anyway and it can only be confusing and take up space.

Comment (stevenroose) [reply sample-lnd.conf]: I would remove 
- debughtlc
- hodlhtlc
- simnet
- regtest
to be honest.

Comment (Roasbeef) [reply sample-lnd.conf]: `simnet` and `regtest` are fine as developers may be using that locally for testing purposes. It's just the two HTLC modifiers that are only used within our automated integration tests

**Comment by stevenroose** [comment]
Updated.

**Comment by stevenroose** [comment]
It looks like the conf file is not really parsed as TOML.. In my setup it seems to require the section prefix explicitly.
```toml
[Bitcoin]
active=1
```
Returns `either bitcoin.active or litecoin.active must be set to 1 (true)`

While 
```toml
[Bitcoin]
bitcoin.active=1
```
works.

Is this a TOML parsing problem or does it look for a command line flag explicitly? (Can look into this tonight after work.)

**Comment by Roasbeef** [comment]
That's because we add the `namespace` struct stags to each of the fields as well. This is what enables users to do `--bitcoin.active` on the command line.

**Comment by stevenroose** [comment]
In TOML, the `[header]` should determine the "namespace", no? Let me look into this tonight.

**Pull request merged**